tensorflow/lite/toco/graph_transformations/hardcode_min_max.cc

```diff
@@ -430,6 +430,7 @@ ::tensorflow::Status HardcodeMinMax::Run(Model* model, std::size_t op_index,
       changed = HardcodeMinMaxForAverageOrMaxPool(model, op);
       break;
 
+    case OperatorType::kAbs:
```

Comment:
is this hardcode case intended? This hardcodes the min/max of the output to the min/max of the input. this means if the abs input is [-2, -1] than the output range will be the same, which seems off. 

Couple options:
(1) Choose to not have a hardcode case here, instead relying on a Fakequant from contrib/quatnize?
(2) Hardcode to:
r = max(abs(input_min), (input_max))
[-r, r]

WYDT?

Reply:
Yes, it **was** intended. Like mentioned https://github.com/tensorflow/tensorflow/pull/31914#issuecomment-524169853, I have been choosing whether to let input/output of Abs having different quantization representation. But I really missed the value range that you pointed out. This is not what we want.

Reply:
Removed Abs from hardcode, maybe we can learn min/max in further patch.